config/ecs/container-insights/otel-agent-default-config.yaml

```diff
@@ -0,0 +1,46 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:55680
+      http:
+        endpoint: localhost:55681
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
+  awsecscontainermetrics:
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+  batch/metrics:
+    timeout: 60s
+
+exporters:
+  awsxray:
+  awsemf/application:
+    log_group_name: aws/ecs/containerinsights/{aws.ecs.cluster.name}/application/metrics
+    log_stream_name: { aws.ecs.task.id }
+  awsemf/performance:
+    log_group_name: /aws/ecs/containerinsights/{aws.ecs.cluster.name}/performance
+    log_stream_name: { aws.ecs.task.id }
+
+service:
+  pipelines:
+    traces:
+      receivers: [otlp,awsxray]
+      processors: [batch/traces]
+      exporters: [awsxray]
+    metrics/application:
+      receivers: [otlp]
+      exporters: [awsemf/application]
+    metrics/performance:
+      receivers: [awsecscontainermetrics ]
+      processors: [ batch/metrics ]
```

Comment:
Do we need this batch processor as default? Same for Trace pipeline. I remember in one weekly Meeting Dennis questioned this. I am not sure why do we need this.

Reply:
I used this from the default config, I don't know why. @mxiamxia Should we remove this?

Reply:
It seems we've done the opposite way. we need to batch in `metrics/application` pipeline. But for the performance metrics I think Rayhan has timer to pull the data in the receiver so we can remove it in `metrics/performance` pipeline.